lua/weapons/weapon_taser.lua

```diff
@@ -24,7 +24,7 @@ if CLIENT then
         if !IsValid(ply) or !IsValid(ply.IsTaseredBy) then return end
 
         local originvec, originang = ply:GetBonePosition(1)
-        ply:ManipulateBonePosition(0, Vector(0, 0, -35))
+        ply:ManipulateBonePosition(0, Vector(0, 0, -math.abs(ply:GetViewOffset().z)))
```

Comment:
For a human, this value is 64, for a pony, I think it's 42. Maybe you want `ply:IsPony() and -13 or -35`